tiatoolbox/models/architecture/nuclick.py

```diff
@@ -119,7 +120,7 @@ def get_block(
                 Size of stride in the convolution layer.
             use_bias (bool):
                 Whether to use bias in the convolution layer.
-            dilation_rates (list):
+            dilation_rate (list):
```

Comment:
I generally think arguments which must be sequences such as a numpy array, tuple, list etc. should be plural to indicate that it is not a single value / scalar.